docs/source/markdown/podman-quadlet-list.1.md

```diff
@@ -0,0 +1,55 @@
+% podman-quadlet-list 1
+
+## NAME
+podman\-quadlet\-list - List installed quadlets
+
+## SYNOPSIS
+**podman quadlet list** [*options*]
+
+## DESCRIPTION
+
+List all Quadlets configured for the current user.
+
+## OPTIONS
+
+#### **--filter**, **-f**
+
+Filter output based on conditions give.
+
+The *filters* argument format is of `key=value`. If there is more than one *filter*, then pass multiple OPTIONS: **--filter** *foo=bar* **--filter** *bif=baz*.
+
+Supported filters:
+
+| Filter     | Description                                                                                      |
+|------------|--------------------------------------------------------------------------------------------------|
+| name       | Filter by quadlet name.                                                                          |
```

Comment:
Can you add a TODO somewhere to add an Application filter to this?

Reply:
Added in code.